src/py/flwr_tool/check_copyright.py

```diff
@@ -65,12 +85,19 @@ def _check_copyright(dir_list: List[str]) -> None:
 
 
 if __name__ == "__main__":
-    if len(sys.argv) == 0:
+
+    args = parser.parse_args()
+
+    if args.include is None:
         raise Exception(  # pylint: disable=W0719
             "Please provide at least one directory path relative "
             "to your current working directory."
         )
-    for i, _ in enumerate(sys.argv):
-        abs_path: str = os.path.abspath(os.path.join(os.getcwd(), sys.argv[i]))
+
+    # Ensure dirs end with "/"
+    exc_dirs = [ex_dir.rstrip("/") + "/" for ex_dir in args.exclude]
+
+    for include_path in args.include:
+        abs_path: str = os.path.abspath(os.path.join(os.getcwd(), include_path))
```

Comment:
```suggestion
        str = str((Path.cwd() / include_path).resolve())
```

Let's use `pathlib.Path` here for consistency and readability.